web-marketplace/src/pages/Post/Post.Header.tsx

```diff
@@ -1,5 +1,6 @@
 import { useState } from 'react';
```

Comment:
missing `COPY_SUCCESS`

Reply:
This constant is currently part of `web-component`.
As we discussed it might be best to avoid lingui usage in `web-components`. So I'll extract it into `web-marketplace`. I'll put it under `lib/constants/sharedConstants.ts`, let me know if you have other suggestions for the file organization.

Reply:
yes that sounds good to me

Reply:
shared.constants.ts could work too as file name to be consistent with the other similar files